scripts/csi_test/csi_tests_to_skip_community_a9k

```diff
@@ -0,0 +1,6 @@
+GetCapacity
+ListVolumes
+ValidateVolumeCapabilities
+volume lifecycle
+ListSnapshots
+NodeGetVolumeStats
```

Comment:
Add a newline at EOF (in all relevant files)

Reply:
done